pages/events/[id].js

```diff
@@ -12,20 +16,88 @@ function EventDetail() {
     return <h1>hmm... no event yet 🤔?</h1>;
   }
 
-  const { title, startTime, endTime, location, creator, isParticipant } =
-    currentEvent;
+  const { title, startTime, endTime, location, creator } = currentEvent;
+
+  function handleVoteEmoji(voteEmoji) {
+    if (emojiCheckmark === "") {
+      setEmojiCheckmark(voteEmoji);
+    } else {
+      setEmojiCheckmark("");
+    }
+  }
 
   return (
-    <>
+    <StyledWrapper>
       <h2>{title}</h2>
-      <span>{startTime}</span>
-      <span>{endTime}</span>
-      <p>Location: {location}</p>
-      <p>Creator: {creator}</p>
-      <p>Are you there? {isParticipant ? "Yes" : "No"}</p>
-      <Link href="/events">Go Back</Link>
-    </>
+      <StyledEmojiCheckmark>{emojiCheckmark}</StyledEmojiCheckmark>
+      <StyledCardContainer>
+        <span>{startTime}</span>
+        <span>{endTime}</span>
+        <p>Location: {location}</p>
+        <p>Creator: {creator}</p>
+      </StyledCardContainer>
+      <StyledVoteButtonContainer>
+        {["👍", "❓", "👎"].map((voteEmoji) => {
+          return (
+            <StyledVoteButtons
+              onClick={() => handleVoteEmoji(voteEmoji)}
+              key={voteEmoji}
+            >
+              <span aria-label="emoji">{voteEmoji}</span>
+            </StyledVoteButtons>
+          );
+        })}
+      </StyledVoteButtonContainer>
+      <Link href="/events">
+        <StyledLink>↩︎</StyledLink>
+      </Link>
+    </StyledWrapper>
   );
 }
 
-export default EventDetail;
+const StyledEmojiCheckmark = styled.span`
+  position: absolute;
+  top: 61px;
+  left: 275px;
+  right: 80px;
+  font-size: 27px;
+`;
+
+const StyledVoteButtons = styled.button`
+  font-size: 35px;
+`;
+
+const StyledWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 auto 12px auto;
+`;
+
+const StyledLink = styled.a`
+  display: flex;
+  background-color: lightblue;
+  border: 1px solid;
+  padding: 10px;
+  margin: 50px 170px;
+  font-size: 25px;
+  border-radius: 10px;
+`;
```

Comment:
you can extend the `Link` component:

```suggestion
const StyledLink = styled(Link)`
  display: flex;
  background-color: lightblue;
  border: 1px solid;
  padding: 10px;
  margin: 50px 170px;
  font-size: 25px;
  border-radius: 10px;
`;
```